kuma/payments/jinja2/payments/management.html

```diff
@@ -1,125 +1,23 @@
-                            <td>02/19</td>
-                        </tr>
-                    </tbody>
-                </table>
-                {% if not cancel_request %}
-                    <form id="id_stripe_management_form" method="post">
-                        {% csrf_token %}
-                        <button class="button negative" type="button" toggle-confirmation>{{_('Cancel your subscription')}}</button>
-                        <div id="delete-confirmation" class="hidden" aria-hidden="true">
-                            <div class="confirmation column-8 alert danger">
-                                <h5>{{_('Are you sure?')}}</h5>
-                                <p>
-                                    {{_('You will have to set up a new subscription if you wish to resume making payments to MDN Web Docs.')}}
-                                </p>
-                                <div class="button-group">
-                                    <button class="button" type="button" toggle-confirmation>{{_('Cancel')}}</button>
-                                    <button id="id_stripe_cancel_subscription" name="stripe_cancel_subscription" class="button negative" type="submit">{{_('Confirm')}}</button>
-                                </div>
-                            </div>
-                        </div>
-                    </form>
-                {% endif %}
-            </section>
-        {% else %}
-            <section>
-                <h3>{{_('Monthly Subscription')}}</h3>
-                <div class="bordered-info">
-                    <span>
-                        {% trans url=settings.SITE_URL + url('payments_index') %}
-                            You have no active subscriptions.
-                            Why not <a href="{{ url }}">set one up</a>?
-                        {% endtrans %}
-                    </span>
-                </div>
-            </section>
-        {% endif %}
-        {% if cancel_request %}
-            {% if cancel_success %}
-                <div class="confirmation column-8 alert success">
-                    <b>{{_('Your monthly subscription has been successfully canceled')}}</b>
-                </div>
-            {% else %}
-                <div class="confirmation column-8 alert danger">
-                    <b>
-                        {% trans url=support_mail_link, email=support_mail, reference_number=request.user.stripe_customer_id %}
-                            There was a problem canceling your subscription,
-                            please contact <a href="{{ url }}">{{ email }}</a>
-                            and provide your customer reference number {{ reference_number }}.
-                        {% endtrans %}
-                    </b>
-                </div>
-            {% endif %}
-        {% endif %}
-    </div>
+{% block document_head %}
+    {{ render_react("SPA", request.LANGUAGE_CODE, request.get_full_path(), True)|safe }}
```

Comment:
No need for the support email on the management page??

Reply:
It's removed from the new mockups-- the only time you would see it is if the error screen renders like so (this is the generic error screen on all payments pages):
<img width="635" alt="Screen Shot 2020-04-28 at 9 44 58 PM" src="https://user-images.githubusercontent.com/650/80553925-97eb4000-8999-11ea-88f6-517c5f831c93.png">